packages/sdk/chainDeploy/mainnets/zkevm.ts

```diff
@@ -0,0 +1,121 @@
+import { zkevm } from "@ionicprotocol/chains";
+import { assetSymbols, underlying } from "@ionicprotocol/types";
+import { ethers } from "ethers";
+
+import {
+  ChainDeployConfig,
+  deployAlgebraPriceOracle,
+  deployBalancerRateProviderPriceOracle,
+  deployPythPriceOracle
+} from "../helpers";
+import { BalancerRateProviderAsset, ConcentratedLiquidityOracleConfig, PythAsset } from "../helpers/types";
+
+const assets = zkevm.assets;
+
+export const deployConfig: ChainDeployConfig = {
+  wtoken: zkevm.chainAddresses.W_TOKEN,
+  nativeTokenUsdChainlinkFeed: "",
+  nativeTokenName: "Ethereum",
+  nativeTokenSymbol: "ETH",
+  stableToken: zkevm.chainAddresses.STABLE_TOKEN,
+  wBTCToken: zkevm.chainAddresses.W_BTC_TOKEN,
+  blocksPerYear: zkevm.specificParams.blocksPerYear.toNumber(),
+  uniswap: {
+    hardcoded: [],
+    uniswapData: [],
+    pairInitHashCode: ethers.utils.hexlify("0x1f475d88284b09799561ca05d87dc757c1ff4a9f48983cdb84d1dd6e209d3ae2"),
+    uniswapV2RouterAddress: "0xbdFa4a05372a10172EeEB75075c85FCbff521625",
+    uniswapV2FactoryAddress: "",
+    uniswapV3FactoryAddress: "0x865412B6cDf424bE36088fE3DeC2A072a26Cc494",
```

Comment:
0x865412B6cDf424bE36088fE3DeC2A072a26Cc494 either

Reply:
yeah, there is actually neither deployed. Will set them to address zero